### PR TITLE
RHDEVDOCS 5945 Clarify the need for Tekton Results for statistics

### PR DESCRIPTION
**Pull request by mramendi**
Version(s):

please cp to pipelines-docs-1.14

Issue:

RHDEVDOCS 5945

Link to docs preview:

https://72359--docspreview.netlify.app/openshift-pipelines/latest/about/op-release-notes#new-features-1-14_op-release-notes
https://72359--docspreview.netlify.app/openshift-pipelines/latest/create/working-with-pipelines-web-console#op-console-statistics_working-with-pipelines-web-console

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by ppitonak — APPROVED** [review]

File: modules/op-console-statistics.adoc [hunk 1]
```diff
@@ -7,7 +7,10 @@
 
 You can view statistics related to execution of pipelines in the web console.
 
-To view the statistic information, you must first enable the {pipelines-shortname} console plugin.
+To view the statistic information, you must complete the following steps:
+
+* Install {tekton-results}
```
Comment (mburke5678) [inline modules/op-console-statistics.adoc:12]: Do we want to add something like: For more infromation, see _Using Tekton Results for OpenShift Pipelines observability_ in the _Additional resources_ section.

Give folks some sense of where to go to install it.

**Comment by mburke5678** [comment]
@mramendi Just one random thought. Otherwise LGTM

File: modules/op-console-statistics.adoc [hunk 2]
```diff
@@ -7,7 +7,10 @@
 
 You can view statistics related to execution of pipelines in the web console.
 
-To view the statistic information, you must first enable the {pipelines-shortname} console plugin.
+To view the statistic information, you must complete the following steps:
+
+* Install {tekton-results}. For more information about installing {tekton-results}, see _Using Tekton Results for OpenShift Pipelines observability_ in the _Additional resources_ section.
+* Enable the {pipelines-shortname} console plugin
```
Comment (kalexand-rh) [inline modules/op-console-statistics.adoc:13]:
```suggestion
* Enable the {pipelines-shortname} console plugin.
```

**Review by aireilly — APPROVED** [review]
lgtm

**Pull request merged**